examples/rl/platoon/train/reward.py

```diff
@@ -59,13 +59,10 @@ def step(self, action):
     def _reward(self, obs, env_reward):
         reward = {agent_id: np.float64(0) for agent_id in obs.keys()}
 
-        leader_name = "Leader-007"
         leader = None
         max_agent_steps_completed = 0
         for agent_id, agent_obs in obs.items():
-            neighbor_vehicles = _get_neighbor_vehicles(
-                obs=agent_obs, neighbor_name=leader_name
-            )
+            neighbor_vehicles = _get_leader(obs=agent_obs)
```

Comment:
I think `_get_leader()` is a misleading name for the method. It should indicate plurality and also what it is getting. I also think that `neighbor_vehicles` is probably not specific enough.

```suggestion
            interest_vehicles = _get_interest_vehicles(obs=agent_obs)
```

Reply:
I have changed it accordingly.